.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,6 +18,16 @@ can uncomment the below line to indicate which issue your PR fixes, for example
 Fixes #
 -->
 
+### Testing done
+<!--
+How has this code been tested?
+
+Before reviewers can be confident in the correctness of a changeset,
+it needs to be tested. In this section, briefly describe the testing
+that has already done or which is planned. The testing should be enough
+to help people feel comfortable in the changeset's correctness.
```

Comment:
Is there value in both opening by stating that testing helps reviewers feel comfortable with a change's correctness, and closing by stating that the tests should help reviewers feel comfortable with a change's correctness?